README.md

```diff
@@ -12,9 +12,10 @@ This repository contains the ZKsync Solidity compiler.
 ## System Requirements
 
 Supported platforms:
-- **Linux: x86_64**
-   * MUSL-based static builds do not depend on system libraries and can be run on any recent Linux distribution.
-- **MacOS 11+: x86_64, arm64 (Apple silicon)**
+- **Linux: x86_64, ARM64**
+   * [MUSL](https://musl.libc.org)-based static builds are currently supported, but may be deprecated in the future; they do not depend on system libraries and can be run on any recent Linux distribution.

```

Comment:
Our GNU libraries are linked statically so they don't depend on system libraries either.
The only reason we still have MUSL is the old versions of our Hardhat plugin which will be downloading MUSL even if you have specified the latest version of `zksolc` in the project config.

Reply:
we're simultaneously wrote on the same topic (https://github.com/matter-labs/era-compiler-solidity/pull/124#discussion_r1716526527) :)

Reply:
1ea0a63 + bd52869